### DISCUSSION TITLE
PROXY

### DISCUSSION THREAD

**Discussion by ghost** [Tribler/tribler#7079] [Q&A]
If I specify my proxy in the "Network" tab and then right-click on the torrent and select no proxy - will it be loaded without a proxy or through my personal proxy?

**Comment by absolutep** [comment]
If you are talking about setting up this
![Untitled](https://user-images.githubusercontent.com/97702149/197132780-90ab62c9-cde7-49d9-83c9-ee09e4a3b84d.png)


Then, I think everything will go through the proxy settings, you will specify in the image above irrespective of your hop settings. But maybe you can ping developer here for more technical clarifications.
